ouroboros-network/sim-tests-lib/Test/Ouroboros/Network/Diffusion/Node/MiniProtocols.hs

```diff
@@ -258,26 +255,23 @@ applications debugTracer nodeKernel
                , aaShouldChainSyncExit
                , aaChainSyncEarlyExit
                , aaOwnPeerSharing
-               , aaUpdateOutboundConnectionsState
                }
              toHeader =
-    Diff.Applications
-      { Diff.daApplicationInitiatorMode =
+    Common.Applications
```

Comment:
Keep the original qualification (so it's easier for us to rebase over this PR).

Reply:
Same as other comments